### PR TITLE
add csdn.net, jianshu..com

### PR DESCRIPTION
**Pull request by AustinZhang1024**
add two website in moblie to remove ads, and annoying ads

### CODE REVIEW

**Comment by Alex-302** [comment]
@OdinZhang Hi. Please describe what rules block(If it doesn't make it difficult - with screenshots).
I see you added all tyles of rules to Mobile Ads filter. Subscriptions and annoying elements must be added to Annoyances filter.

**Comment by OdinZhang** [comment]
these rules can auto expand the hidding element and remove the dialog which guide users to there app.
before:
Screenshot_20210731-124859_Edge
Screenshot_20210731-124952_Edge
after:
Screenshot_20210731-124914_Edge
Screenshot_20210731-125008_Edge

File: MobileFilter/sections/specific_app.txt [hunk 1]
```diff
@@ -812,5 +812,3 @@ c0server.jags.co.jp/jss/img/banners/$app=jp.co.jags.android.Bokumaka
 ||log.kajicam.com^
 ! Google Maps Promoted Pins
 ||mobilemaps-pa.googleapis.com/google.internal.mothership.maps.mobilemaps.ads.v1.MobileMapsAdsService/ListPromotedPinAds^
-! WeChat
-||extshort.weixin.qq.com/mmtls^$app=com.tencent.mm
```
Comment (Alex-302) [inline MobileFilter/sections/specific_app.txt]: Why removed?

Comment (OdinZhang) [reply MobileFilter/sections/specific_app.txt]: this rule may affect the other features of this app

File: AnnoyancesFilter/sections/annoyances.txt [hunk 2]
```diff
@@ -4409,3 +4409,16 @@ post-gazette.com#$##toolbar > #share-bar { display: none!important; }
...
+blog.csdn.net#%#//scriptlet('remove-attr', 'style', '.article_content')
+blog.csdn.net##.btn_mod
+blog.csdn.net##.readall_box
+blog.csdn.net##.feed-Sign-span
+blog.csdn.net##.weixin-shadowbox
+jianshu.com##.call-app-btn
+jianshu.com##.collapse-tips
+jianshu.com#%#//scriptlet('remove-class', 'collapse-free-content')
+jianshu.com##.download-app-guidance
+jianshu.com#$#body{overflow:auto!important}
```
[the remark this answers is not in the document]
Comment (OdinZhang) [reply AnnoyancesFilter/sections/annoyances.txt]: all changed

[the remark this answers is not in the document]
Comment (Alex-302) [reply AnnoyancesFilter/sections/annoyances.txt]: If you hiding popup which blocks scrolling, all rules for this popup(popup hiding and scroll unblocking) must be CSS. In other case element hiding rules will break the site on iOS.

[the remark this answers is not in the document]
Comment (OdinZhang) [reply AnnoyancesFilter/sections/annoyances.txt]: all done

File: AnnoyancesFilter/sections/annoyances.txt [hunk 3]
```diff
@@ -4437,3 +4437,16 @@ post-gazette.com#$##toolbar > #share-bar { display: none!important; }
...
 ||mayu-pravo.com/share/slider/
 ||u.today/img/satoshi-twitter.png
+blog.csdn.net#$#.btn_open_app_prompt_div { display: none!important; }
+blog.csdn.net#$#.article_content { overflow: auto !important; height: auto !important; }
+blog.csdn.net#$#.btn_mod { display: none!important; }
+blog.csdn.net#$#.readall_box { display: none!important; }
+blog.csdn.net#$#.feed-Sign-span { display: none!important; }
+blog.csdn.net#$#.weixin-shadowbox { display: none!important; }
+jianshu.com#$#.call-app-btn { display: none!important; }
+jianshu.com#$#.collapse-tips { display: none!important; }
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt:4447]: Hi. All these elements used in popups, which block scrolling?

Comment (OdinZhang) [reply AnnoyancesFilter/sections/annoyances.txt:4447]: maybe I misunderstanded, `blog.csdn.net#$#.article_content { overflow: auto !important; height: auto !important; }` and `jianshu.com#$#.collapse-free-content { overflow: auto !important; height: auto !important; }` is for scrolling and the others are normal HTML elements, just need to hide them or remove.

Comment (Alex-302) [reply AnnoyancesFilter/sections/annoyances.txt:4447]: Ok. Will try to explain)
For examle popup has three elements:
- popup content;
- overlay/bacground;
- style for scrolling blocking.
In this case you need thee rules:
- 2x `{ display: none!important; }`;
- 1x `{ overflow: auto !important; }`(or changed style for `position`, which used on some sites to block scrolling).
All other elements can be hidden by `##`.

Comment (OdinZhang) [reply AnnoyancesFilter/sections/annoyances.txt]: I think this time the rules are right.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**